src/Umbraco.Web/Editors/BackOfficeController.cs

```diff
@@ -32,32 +33,32 @@ public class BackOfficeController : UmbracoController
         private BackOfficeOwinUserManager _userManager;
         private BackOfficeSignInManager _signInManager;
         private readonly IUmbracoVersion _umbracoVersion;
-        private readonly IContentSettings _contentSettings;
+        private readonly ContentSettings _contentSettings;
         private readonly IHostingEnvironment _hostingEnvironment;
-        private readonly IRuntimeSettings _runtimeSettings;
-        private readonly ISecuritySettings _securitySettings;
+        private readonly RuntimeSettings _runtimeSettings;
+        private readonly SecuritySettings _securitySettings;
 
         public BackOfficeController(
             UmbracoFeatures features,
-            IGlobalSettings globalSettings,
+            IOptions<GlobalSettings> globalSettings,
             IUmbracoContextAccessor umbracoContextAccessor,
             ServiceContext services,
             AppCaches appCaches,
             IProfilingLogger profilingLogger,
             IUmbracoVersion umbracoVersion,
-            IContentSettings contentSettings,
+            IOptions<ContentSettings> contentSettings,
             IHostingEnvironment hostingEnvironment,
-            IRuntimeSettings settings,
-            ISecuritySettings securitySettings)
+            IOptions<RuntimeSettings> settings,
+            IOptions<SecuritySettings> securitySettings)
             : base(globalSettings, umbracoContextAccessor, services, appCaches, profilingLogger)
 
         {
             _features = features;
             _umbracoVersion = umbracoVersion;
-            _contentSettings = contentSettings ?? throw new ArgumentNullException(nameof(contentSettings));
+            _contentSettings = contentSettings.Value;
```

Comment:
This is part of `Umbraco.Web`. Most of this class is already migrated (e.g the constructor). So this will never be executes and is mostly just to allow tests to pass